src/styles/App.css

```diff
@@ -0,0 +1,52 @@
+body {
+background-color: #282c34;
+}
+
+.App {
+  text-align: center;  
+  color: beige;
+}
+
+header {
+  /* color: azure; */
+  height:50vh;
+}
+
+main {
+  /* height: 45vh; */
+  /* background-color: black; */
+  
+  /* background-image: url("./../assets/images/alberta-background.jpg");
+  background-repeat: no-repeat;
+  background-size: contain; */
+}
+
+main p {
+  margin: 20px;
+}
+
+footer {
+  /* height: 10vh; */
+}
+
+.wrapper {
+  display: flex;
+  flex-direction: column;
+
+  width: 80%;
+  margin: 0 auto;
+}
+
+.center {
+  text-align: center;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+
+@media (min-width: 400px) {
+  header {
+    /* color: azure; */
+    /* height: 50vh; */
```

Comment:
Not sure if this is still being used, but if not then the commented code can be removed